macOS/Configuration/App/Sparkle.xcconfig

```diff
@@ -16,8 +16,10 @@
 // URLs for various build configuration should be kept here in the format of
 // SPARKLE_URL_<CONFIG> where <CONFIG> stands for the build configuration.
 // Release automation is looking for URLs here following that format.
-SPARKLE_URL_RELEASE = "https://staticcdn.duckduckgo.com/macos-desktop-browser/appcast2.xml"
-SPARKLE_URL_ALPHA = "https://staticcdn.duckduckgo.com/macos-desktop-browser/appcast-alpha.xml"
+// `/$()/` is a workaround for adding 2 forward slashes so that they aren't
+// interpreted as a comment.
+SPARKLE_URL_RELEASE = https:/$()/staticcdn.duckduckgo.com/macos-desktop-browser/appcast2.xml
+SPARKLE_URL_ALPHA = https:/$()/staticcdn.duckduckgo.com/macos-desktop-browser/alpha/appcast-alpha.xml
```

Comment:
This is an important change because of which #1363 can’t be merged as the Sparkle feed URL is broken there.